src/redux/delete-reservation/deleteReservationSlice.js

```diff
@@ -0,0 +1,65 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'http://localhost:4000';
+
+export const fetchReservations = createAsyncThunk('bookings/fetchReservations', async (_, { rejectWithValue }) => {
+  try {
+    const response = await axios.get(`${url}/bookings`, { withCredentials: true });
+    return response.data;
+  } catch (error) {
+    return rejectWithValue(error.response.data);
+  }
+});
+
+export const deleteReservation = createAsyncThunk('bookings/deleteReservation', async (userId, { rejectWithValue }) => {
+  try {
+    const response = await axios.delete(`${url}/bookings/${userId}`, { withCredentials: true });
+    return response.data;
+  } catch (error) {
+    return rejectWithValue(error.response.data);
+  }
+});
```

Comment:
- [ ] Excellent job implementing the async thunk action to delete `reservation`  However, Kindly ensure that the user can delete the reservation. Currently, the reservation delete feature is not working.
`Hint:` 
```
axios.delete(`${url}/bookings/${booking_id}`, { place your headers here });
```